lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -65,7 +70,31 @@
                 selectText: function(event) {
                     event.preventDefault();
                     $(event.currentTarget).select();
-                }
+                },
+
+                leaveTeam: function (event) {
+                    event.preventDefault();
+                    var view = this;
+                    $.ajax({
+                       type: 'DELETE',
+                       url: view.teamsMembershipDetailUrl.replace('team_id', view.model.get('id'))
+                    }).done(function (data) {
+                       view.model.fetch({});
+                    }).fail(function (data) {
+                       try {
+                           var errors = JSON.parse(data.responseText);
+                           view.showMessage(errors.user_message);
+                       } catch (error) {
+                           view.showMessage(view.errorMessage);
+                       }
+                    });
+                },
+
+                showMessage: function (message) {
```

Comment:
Can we move this method to teamUtils, as we are already using this in join team too.